.travis.yml

```diff
@@ -53,13 +53,12 @@ script:
       make html;
       cd ..;
       doctr deploy --deploy-repo xonsh/xonsh-docs --gh-pages-docs dev;
+      git checkout $(git describe --tags `git rev-list --tags --max-count=1`);
+      cd docs;
+      make html;
```

Comment:
You might want to put a `make clean` here to clear out any cruft from the dev build above.